src/ocean/math/WideUInt.d

```diff
@@ -0,0 +1,591 @@
+/*******************************************************************************
+
+    Implementation struct emulating positive number with value range exceeding
```

Comment:
This is clearer:
"Struct emulating a large (bigger than ulong) non-negative integer of fixed range (defined via template argument)."